client/boot/project/wordpress-com.js

```diff
@@ -85,7 +87,11 @@ export function setupMiddlewares( currentUser, reduxStore ) {
 	// Render Layout only for non-isomorphic sections.
 	// Isomorphic sections will take care of rendering their Layout last themselves.
 	if ( ! document.getElementById( 'primary' ) ) {
-		renderLayout( reduxStore );
+		page( '*', function( context, next ) {
```

Comment:
Thinking through all the different cases sounds great, but I am still a bit confused – why do we need to use `page( '*'…` at all?